mergify_engine/github_types.py

```diff
@@ -472,6 +478,24 @@ class GitHubCheckRun(typing.TypedDict):
     status: GitHubCheckRunStatus
     output: GitHubCheckRunOutput
     conclusion: typing.Optional[GitHubCheckRunConclusion]
+    started_at: ISODateTimeType
+    completed_at: ISODateTimeType
+    html_url: str
+    details_url: str
+    check_suite: GitHubCheckRunCheckSuite
+
+
+class GitHubCheckRunLight(typing.TypedDict):
```

Comment:
Stupid question but how "light" it is? I don't see much data being removed?

Reply:
A lot !!!! (btw, I plan will rename it `CachedGitHubCheckRun`)

The checks["app"] has always the same size and contains le owner details, the permissions details and some of its settings we don't care (I would say 30-40 keys)

The one that can be really big is `checks["output"]["summary"]` it's still present for our check-runs, but we drop it for other CIs.

Reply:
If your plan for this PR or a next one?